src/Parser/grammar.pp

```diff
@@ -0,0 +1,90 @@
+%skip   space               [\x20\x09\x0a\x0d]+
+%skip   doc_                [/**]
+%skip   _doc                [*/]
+%skip   star                [*]
+
+%token  at                  @                           -> annot
+%token  text                [^@].*
+
+%token  annot:identifier    [\\a-zA-Z_][\\a-zA-Z0-9_]*  -> values
```

Comment:
Should be ``\\?[a-zA-Z_][a-zA-Z0-9_](\\[a-zA-Z_][\\a-zA-Z0-9_])*`` actually. Any namespace separator must be followed by a non-digit char (and not by a second separator).
And btw, this regex is more restrictive that the regex of valid PHP class name. Is it on purpose to forbid unicode chars in class names of annotations ?